### ISSUE TITLE
Mi Flora plant sensor: no battery stats?

### ISSUE THREAD

**Issue by thatso** [AlCalzone/ioBroker.ble#106]
I'm currently on BLE 0.7.0 (still waiting for 0.7.2 to be released with the additional MAC for hygrometers) and don't see battery stats for my plant sensor (MAC c4:7c:8d:xx:xx:xx, firmware 3.2.1) even after two weeks of the initial connection.
Am I missing something or is this a bug?

**Comment by AlCalzone** [comment]
If I remember correctly, the battery information requires an active connection, which is something the adapter does not support yet.

**Comment by thatso** [comment]
Do you mean polling the device with "active connection"?
It seems the neighboring project is doing it this way: [https://www.home-assistant.io/components/miflora/](https://www.home-assistant.io/components/miflora/)

**Comment by AlCalzone** [comment]
Yes, that's what I mean. The problem isn't that I don't know how to do it - its finding the time to do so.

**Comment by capitaenz** [comment]
I would also be very happy about the enhancement of the battery capacity.

**Comment by Klugschieter** [comment]
Yeah I agree, that would be really nice to have.
I have a smart greenhouse in my garden, but it is 15km away from here. Sometimes I don´t go there for 2 weeks,
When in this time the battery of one of the 20 sensors is empty, it is sometimes a little mess, especially when you went there a day ago.
But, no hurry!
I am very thankful of all the work and time you are spending on this great adapter until now!
And in between I can use my MobileApp to check the battery status.

**Comment by eddKv** [comment]
The miflora js on github uses characteristics to read the data. I think this could help, because it looks simple.

**Comment by nukleuz80** [comment]
Is it meanwhile possible to get the battery status?
I installed the adapter yesterday but i don´t get the battery status of MiFlora...

In the past i used this on RPI3+ to fetch the data incl. battery:
https://tutorials-raspberrypi.de/raspberry-pi-miflora-xiaomi-pflanzensensor-openhab/

**Comment by AlCalzone** [comment]
No. https://github.com/AlCalzone/ioBroker.ble/issues/106#issuecomment-487387111 still holds true. I don't find much time to do anything else outside Z-Wave and what I need for it.

**Comment by moeff** [comment]
bump - hope there is any chance to get the feature

atm - i've written a workaround script- to check that data comes in, if not (1day) - it informs me to replace the battery - i use the flower cars for controlling 'leitwert in German' on my aquariums ;)
